kategory-docs/docs/docs/datatypes/ior/README.md

```diff
@@ -42,11 +42,11 @@ val both = ("Warning" to 3).bothIor()
 ```
 
 
-When we look at the `Monad` or `Applicative` instances of `Ior`, we can see that they actually requires a `Semigroup` instance on the left side.
-This is because `Ior` will actually accumulate failures on the left side, very similar to how the [`Validated`](/docs/datatypes/validated) data type does.
+When we look at the `Monad` or `Applicative` instances of `Ior`, we can see that they actually require a `Semigroup` instance on the left side.
+This is because `Ior` will actually accumulates failures on the left side, very similarly to how the [`Validated`](/docs/datatypes/validated) data type does.
```

Comment:
Noo this was correct, you were right!

Reply:
Woops, i changed it to see how it looks like and forgot to remove xD Fixed now!